Remotion/Web/Core/Res/Themes/NovaViso/HTML/FormGrid.css

```diff
@@ -1,105 +1,84 @@
 /*  Style applied to the form grid table's outer structure  */
-table.formGridTable
+.formGridTable
 {
   border-collapse: collapse;
-  border-width: 0;
   width: 100%;
+  background-color: var(--color-standard-background);
 }
 
-td.formGridTitleCell,
-td.formGridSubTitleCell
+.formGridTitleCell,
+.formGridSubTitleCell
 {
   font-weight: bold;
   font-size: 125%;
-  padding: 2px 2px 2px 2px;
-  border: none;
-}
-
-td.formGridTitleCell h1
-{
-  width: 100%;
-  border-bottom: solid 1px #AAAAAA;
-  color: Black;
+  padding: calc(var(--remotion-themed-spacing) * 2) 0;
 }
 
 /*  Style applied to the cells containing the labels  */
-td.formGridLabelsCell
+.formGridLabelsCell
 {
-  background-color: #F0F0F0;
-  text-align: right;
-  padding: 0.3em 1em 0.3em 1em;
-  border-top: none;
-  border-left: none;
-  border-right: none;
-  border-bottom: solid 1px #C8C8C8;
-  width: 17%;
+  box-sizing: border-box;
+  width: 5%;
+  padding-top: var(--remotion-themed-spacing);
+  padding-right: var(--remotion-themed-spacing);
+  padding-bottom: var(--remotion-themed-spacing);
+  padding-left: 0;
```

Comment:
We need this to be smarter. It's possible to have extra columns to the left of the labels columns, it's just never used. So, we shold condition this to it being the first TD in the TR, otherwise the padding is set to spacing.

Reply:
I solved it by setting `padding-right` for each `td` that is not the `:last-child` and then added `padding-left` if it's not `:first-child` (or `:only-child`, such as `formGridTitleCell`, for example). This way, each `td` gets both `padding-left` and `padding-right` except for the first `td` (only `padding-left`) and the last `td` (only `padding-right`). I also set the value to `--remotion-themed-spacing-half` instead of `--remotion-themed-spacing` so that the `padding-left` and `padding-right` of two neighboring `td`s aggregate to a "full" padding, thus avoiding that the space between two `td`s doesn't get too wide.

Reply:
sounds reasonable.